compiler/main/DynFlags.hs

```diff
@@ -876,7 +877,11 @@ data DynFlags = DynFlags {
 
   -- | Unique supply configuration for testing build determinism
   initialUnique         :: Int,
-  uniqueIncrement       :: Int
+  uniqueIncrement       :: Int,
+
+  -- | Max size, in words, of fields that can be unpacked
+  unboxSmallStrictFields :: Maybe Int,
+  unboxSmallStrictSums   :: Maybe Int
```

Comment:
Yes. There are three possibilities:

(1) The user did not specify the flag at all. (`Nothing`)
(2) The user specified the flag with no argument (default value, e.g., `Just 1`)
(3) The user specified the flag with an argument (`Just n`)